### PR TITLE
🌐 Add Turkish translation for `docs/tr/docs/how-to/index.md`

### PR DESCRIPTION
**Pull request by hasansezertasan**
🌐 Add Turkish translation for `docs/tr/docs/how-to/index.md`

[Original File](https://github.com/tiangolo/fastapi/blob/master/docs/en/docs/how-to/index.md)

Discussion: #9193

### CODE REVIEW

**Comment by hasansezertasan** [comment]
@alperiox, @OzgunCaglarArslan @SametEmin, @mertssmnoglu, can you please leave a review?

**Comment by tiangolo** [comment]
📝 Docs preview for commit 22d4026c349bc147407a311ef32c4cf9cd9e476c at: https://400d59d2.fastapitiangolo.pages.dev

**Review by OzgunCaglarArslan — APPROVED** [review]

**Comment by hasansezertasan** [comment]
Thank you @OzgunCaglarArslan for the review!

File: docs/tr/docs/how-to/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Nasıl Yapılır - Tarifler
+
+Burada çeşitli konular hakkında farklı tarifler veya "nasıl yapılır" kılavuzları yer alıyor.
+
+Bu fikirlerin büyük bir kızmı aşağı yukarı **bağımsız** olacaktır ve çoğu durumda **projenize** hitap ediyorsa bunları incelemelisiniz.
```
Comment (alperiox) [inline docs/tr/docs/how-to/index.md:5]:
```suggestion
Bu fikirlerin büyük bir kısmı aşağı yukarı **bağımsız** olacaktır, çoğu durumda bunları sadece **projenize** hitap ediyorsa incelemelisiniz. 
```

File: docs/tr/docs/how-to/index.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+Projeniz için ilginç ve yararlı görünen bir şey varsa devam edin ve inceleyin, aksi halde bunları atlayabilirsiniz.
+
+!!! tip "İpucu"
+
+    **FastAPI**'ı düzgün (ve önerilen) bir şekilde öğrenmek istiyorsanız [Öğretici - Kullanıcı Rehberi](../tutorial/index.md){.internal-link target=_blank}'ni bölüm bölüm okuyun.
```
Comment (alperiox) [inline docs/tr/docs/how-to/index.md:11]:
```suggestion
    **FastAPI**'ı düzgün (ve önerilen) şekilde öğrenmek istiyorsanız [Öğretici - Kullanıcı Rehberi](../tutorial/index.md){.internal-link target=_blank}'ni bölüm bölüm okuyun.
```

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5bb2d7d8db00f7ddd53dd6e4071a70f7d9359f75 at: https://570df147.fastapitiangolo.pages.dev

**Review by alperiox — APPROVED** [review]

**Comment by SametEmin** [comment]
Looks very good!

**Pull request merged**